pkg/kn/commands/source/binding/describe.go

```diff
@@ -93,12 +93,12 @@ func writeSinkBinding(dw printers.PrefixWriter, binding *v1alpha2.SinkBinding, p
 
 func writeSink(dw printers.PrefixWriter, namespace string, sink *duckv1.Destination) {
 	subWriter := dw.WriteAttribute("Sink", "")
-	if sink.Ref.Namespace != "" && sink.Ref.Namespace != namespace {
-		subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
-	}
-	subWriter.WriteAttribute("Name", sink.Ref.Name)
 	ref := sink.Ref
 	if ref != nil {
+		subWriter.WriteAttribute("Name", sink.Ref.Name)
+		if sink.Ref.Namespace != "" && sink.Ref.Namespace != namespace {
+			subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
```

Comment:
I can comment line 100 and still pass all tests...

```go
func writeSink(dw printers.PrefixWriter, namespace string, sink *duckv1.Destination) {
	subWriter := dw.WriteAttribute("Sink", "")
	ref := sink.Ref
	if ref != nil {
		subWriter.WriteAttribute("Name", sink.Ref.Name)
		if sink.Ref.Namespace != "" && sink.Ref.Namespace != namespace {
			// subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
		}
		subWriter.WriteAttribute("Resource", fmt.Sprintf("%s (%s)", sink.Ref.Kind, sink.Ref.APIVersion))
	}
	uri := sink.URI
	if uri != nil {
		subWriter.WriteAttribute("URI", uri.String())
	}
}
```

Reply:
The namespace for the sink ref `myservicenamespace` as well as the kind `Service ()` have been added to the original test to increase its coverage. 

Curious about the need for `if sink.Ref.Namespace != "" && sink.Ref.Namespace != namespace {` though?

Reply:
> Curious about the need for if sink.Ref.Namespace != "" && sink.Ref.Namespace != namespace { though?

@danielhelfand: We dont support referencing sink in a different namespace yet, the namespace would be same as the one `kn` operating in.
This would be helpful in printing the sink namespaces if the source is created say via yaml or `kn service import` / `kn service apply`(in future).